tests/tck-build-logic/src/main/resources/contributing/questions.json

```diff
@@ -0,0 +1,37 @@
+  },
+  {
+    "question-key": "testsLocation",
+    "question": "Where did you implement tests for this library? Absolute path to the directory containing java packages where you implemented tests: ",
+    "help": "An absolute path to the directory that contains your tests. This path must be on your system and not some online location. Be aware that for all tests where you are not the sole author, you have to add a comment that proves that you may publish them under the specified license manually"
+  },
+  {
+    "question-key": "resourcesLocation",
+    "question": "Do your tests need any kind of resources? Absolute path to the resources directory required for your tests (type \"-\" if resources are not required): ",
+    "help": "An absolute path to the directory that contains resources required for your tests. This path must be on your system and not some online location. "
+  },
+  {
+    "question-key": "docker",
+    "question": "Do your tests use docker? Enter the next docker image name (to stop type \"-\"): ",
+    "help": "Enter the docker images (press enter after each image name you enter) that you want to use in your tests. Docker image declaration consists of two parts separated with \":\" in the following format: \"imageName:version\". When you finish adding docker images, type \"-\" to terminate the inclusion process"
+  },
+  {
+    "question-key": "allowedPackages",
+    "question": "What package you want to include? Enter the next package (to stop type \"-\")",
+    "help": "Enter the packages (press enter after each package you entered) that you want to include in your metadata. When you finish adding packages, type \"-\" to terminate the inclusion process"
+  },
+  {
+    "question-key": "additionalDependencies",
+    "question": "What are the additional dependencies you want to include? Enter the next dependency (to stop type \"-\")",
+    "help": "Enter the additional dependencies (press enter after each dependency) you want to include use in tests. Provide dependencies in form of Maven coordinates. Maven coordinates consist of three parts in the following format \"groupId:artifactId:version\". For more information visit: https://maven.apache.org/repositories/artifacts.html When you finish adding dependencies, type \"-\" to terminate the inclusion process"
+  },
+  {
+    "question-key": "shouldCreatePullRequest",
+    "question": "Do you want to create a pull request to the reachability metadata repository [Y/n]:",
+    "help": "In case you want to open a pull request to Reachability metadata repository automatically, type \"y\". This way the task will create a new branch for you, commit changes with a proper message, and push the branch remotely. At the end, the task will give you a link to the Github repository where you should fill in a pull request checklist and complete the process."
```

Comment:
```suggestion
    "help": "In case you want to open a pull request to reachability metadata repository automatically, type \"y\". This way the task will create a new branch for you, commit changes with a proper message, and push the branch remotely. At the end, the task will give you a link to the Github repository where you should fill in a pull request checklist and complete the process."
```